security/sanitizers.rst

```diff
@@ -293,14 +293,73 @@ Then, specify this toolchain file as part of your Conan profile:
    compiler.sanitizer=AddressUndefinedBehavior
 
    [conf]
-   tools.cmake.cmaketoolchain:user_toolchain=cmake/my_toolchain.cmake
+   tools.cmake.cmaketoolchain:user_toolchain=["<path_to>/cmake/my_toolchain.cmake"]
 
 This way, you can keep your existing CMake toolchain file and still leverage Conan profiles to manage other settings.
 
 Note that this approach only works if all dependencies are built using CMake and the ``CMakeToolchain`` integration.
 If you have dependencies using other build systems (e.g., Meson, Autotools), those dependencies will not receive the sanitizer flags
 defined in your custom CMake toolchain file.
 
+Managing sanitizers as a custom CMake Build Type
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Similar to managing sanitizers with a custom CMake toolchain, but without ``compiler.sanitizer`` custom settings,
+is to define sanitizers as part of a custom CMake build type. This way, you can select the desired sanitizer configuration
+by specifying the build type during the CMake configuration step.
```

Comment:
For me it's a bit difficult to understand this paragraph as it is, may we change this for something like:


```suggestion
Another option, without using the custom ``compiler.sanitizer`` setting, is to define sanitizers as a custom CMake build type. In this approach, you select the sanitizer configuration by choosing the build type during the CMake configure step.
```